include/tvm/meta_schedule/space_generator.h

```diff
@@ -153,7 +153,7 @@ class SpaceGenerator : public runtime::ObjectRef {
    *  to blocks in post-DFS order.
    * \return The design space generator created.
    */
-  TVM_DLL static SpaceGenerator PostOrderApply();
+  TVM_DLL static SpaceGenerator PostOrderApply(runtime::PackedFunc f_block_filter = nullptr);
```

Comment:
We don't need default value in header file IIRC.

Reply:
I think its a good idea to provide a default value for this function since using a filter is an unusual case. So we need to either set the default here or in `post_order_apply.cc`. My understanding of good c++ hygiene is that defaults should be set in headers when possible.

Reply:
You're right. It's suggested to leave default values in header files. Thanks for the correction.